tools-ui/src/main/java/com/hedera/hashgraph/client/ui/CreatePaneController.java

```diff
@@ -2188,14 +2226,25 @@ private void setupManagedProperty(Node... nodes) {
 		}
 	}
 
+	private void processKey(JsonObject key, ScrollPane keyPane) {
+
+		if (!key.equals(new JsonObject())) {
+			newKeyJSON = key;
+		}
+
+		if (!key.equals(new JsonObject()) && !key.toString().equals("{\"keyList\":{\"keys\":[]}}")) {
```

Comment:
nit: it would be better to have a local var to take the value of `new JsonObject()` to avoid the cost to call the `new` twice.

Reply:
Fixed